CMakeLists.txt

```diff
@@ -557,5 +558,21 @@ add_subdirectory(ffiTestLibrary ${CMAKE_CURRENT_BINARY_DIR}/build/ffiTestLibrary
 add_third_party_dependencies_per_platform()
 
 # Packaging Setup
-
 include(cmake/packaging.cmake)
+
+# GraphViz Setup
+find_program(DOT dot)
+if (DOT)
+    include(cmake/CMakeGraphVizOptions.cmake)
+    if (BUILD_WITH_GRAPHVIZ)
+        add_custom_target(graphviz ALL
+            COMMENT "Build GraphViz Dependency Graph"
+            COMMAND cp ${CMAKE_SOURCE_DIR}/cmake/CMakeGraphVizOptions.cmake .
```

Comment:
1) This line is especially tricky :) It's important to know that the `CMakeGraphVizOptions.cmake` file is searched for in the binary dir. https://cmake.org/cmake/help/latest/module/CMakeGraphVizOptions.html
Maybe we can add a link to the doc?